server/application/terminal.go

```diff
@@ -74,6 +76,9 @@ func (s *terminalHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	userName := sessionmgr.Username(ctx)
+	log.Infof("Serving terminal for userName: %s; podName: %s; container: %s; namespace: %s; cluster: %s", userName, podName, container, namespace, a.ClusterName)
```

Comment:
I think this should probably happen immediately before `newTerminalSession`, after we've validated that this is a valid terminal request.